hrjobcontract/scss/hrjc.scss

```diff
@@ -1,125 +1,128 @@
+  @import 'civihr/modules/variables';
+  @import 'civihr/modules/print';
+  @import 'civihr/modules/fonts';
+  @import 'civihr/modules/angular';
+  @import 'civihr/modules/utils';
+  @import 'civihr/modules/select';
+  @import 'civihr/modules/textangular';
+  @import 'civihr/modules/mixins';
+  @import 'civihr/modules/animate';
 
-  @import "civihr/partials/typography";
-  @import "civihr/partials/global";
+  @import 'civihr/partials/typography';
+  @import 'civihr/partials/global';
   @include global;
-  @import "civihr/partials/badge";
-  @import "civihr/partials/buttons";
-  @import "civihr/partials/collapse";
-  @import "civihr/partials/datepicker";
-  @import "civihr/partials/dropdowns";
-  @import "civihr/partials/forms";
-  @import "civihr/partials/iframe";
-  @import "civihr/partials/modals";
-  @import "civihr/partials/page-contact";
-  @import "civihr/partials/pagination";
-  @import "civihr/partials/panels";
-  @import "civihr/partials/select";
-  @import "civihr/partials/tables";
-  @import "civihr/partials/tooltip";
-  @import "civihr/partials/wells";
+  @import 'civihr/partials/badge';
+  @import 'civihr/partials/buttons';
+  @import 'civihr/partials/collapse';
+  @import 'civihr/partials/dropdowns';
+  @import 'civihr/partials/forms';
+  @import 'civihr/partials/iframe';
+  @import 'civihr/partials/modals';
+  @import 'civihr/partials/page-contact';
+  @import 'civihr/partials/pagination';
+  @import 'civihr/partials/panels';
+  @import 'civihr/partials/select';
+  @import 'civihr/partials/tables';
+  @import 'civihr/partials/tooltip';
+  @import 'civihr/partials/wells';
 
   // CiviHR Job Contract Ext. Custom
   // ==========================================================================
-  @import "hrjc/modules/print";
-  @import "hrjc/modules/animate";
-  @import "hrjc/modules/ui-bootstrap";
+  @import 'hrjc/modules/print';
+  @import 'hrjc/modules/animate';
+  @import 'hrjc/modules/ui-bootstrap';
 
-  @import "hrjc/partials/global";
+  @import 'hrjc/partials/global';
   @include global;
-  @import "hrjc/partials/buttons";
-  @import "hrjc/partials/collapse";
-  @import "hrjc/partials/forms";
-  @import "hrjc/partials/history";
-  @import "hrjc/partials/list";
-  @import "hrjc/partials/loader";
-  @import "hrjc/partials/modals";
-  @import "hrjc/partials/navs";
-  @import "hrjc/partials/panels";
-  @import "hrjc/partials/summary";
-  @import "hrjc/partials/tables";
-  @import "hrjc/partials/wizard";
+  @import 'hrjc/partials/buttons';
+  @import 'hrjc/partials/collapse';
+  @import 'hrjc/partials/forms';
+  @import 'hrjc/partials/history';
+  @import 'hrjc/partials/list';
+  @import 'hrjc/partials/loader';
+  @import 'hrjc/partials/modals';
+  @import 'hrjc/partials/navs';
+  @import 'hrjc/partials/panels';
+  @import 'hrjc/partials/summary';
+  @import 'hrjc/partials/tables';
+  @import 'hrjc/partials/wizard';
+
+  @import '../../../org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
```

Comment:
use SCSSROOT

Reply:
Job Contract has a `compass` based build system and SCSSROOT is not integrated yet, so did not add that.